.drone.star

```diff
@@ -132,8 +132,12 @@ def repo(name, url = "", git = "", sub_path = "", branch = "master", mode = "mak
                 "name": "translation-reader",
                 "image": "owncloudci/transifex:latest",
                 "pull": "always",
+                "environment": {
+                    "NO_INSTALL": "true",
+                },
                 "commands": [
                     "cd '%s'" % work_dir,
+                    'if [ path == "web" ]; then npm install --silent --global --force "$(jq -r ".packageManager" < package.json)"; pnpm config set store-dir ./.pnpm-store; pnpm install; fi',
```

Comment:
Is this a workaround for the ocis-idp frontend not having the packageManager property in package.json?
Shouldn't we just add it there too?

Reply:
No, it is just to install the version specified in package.json in web. Took ref from this PR https://github.com/owncloud/web/pull/9823

Reply:
I'm asking why you check if the folder is web and not something that is less specific, but as you've already merged ... nevermind

Reply:
I guess, since the ci is failing for web only, and similar thing has been done on web recently, just to apply the changes only for the web the condition is there

Reply:
Yes, I'm arguing we should make it more resilient for idp too